### DISCUSSION TITLE
Material Tailwind

### DISCUSSION THREAD

**Discussion by aaron-za-dev** [primefaces/community#1375] [PrimeNG] [answered]
How can I use a style library as a material tailwind  in primeng?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Yes, you can use it, but you may encounter inconsistencies in some cases. To prevent this and provide more flexibility in theming, we are working on the PassThrough and Unstyled mode features. We have already done this for PrimeVue, and you can check it out here: https://tailwind.primevue.org/

A similar situation will also apply to PrimeNG, and we plan to do this for v17.

**Reply by larsm11** [reply]
This is great news, what is your ETA for Passthrough/Unstyled support in PrimeNG?
